bindings/cs/MongoDB.Libmongocrypt/AssemblyInfo.cs

```diff
@@ -1,3 +1,4 @@
 ﻿using System.Runtime.CompilerServices;
 
 [assembly: InternalsVisibleTo("MongoDB.Libmongocrypt.Test")]
+[assembly: InternalsVisibleTo("MongoDB.Libmongocrypt.Test32")]
```

Comment:
This is related to CMAKE magic, as far as I noticed, CMAKE somehow runs tests from `MongoDB.Libmongocrypt.Test` in the `MongoDB.Libmongocrypt.Test**32**` solution which failed since `MongoDB.Libmongocrypt.Test` have tests that use internal classes from `MongoDB.Libmongocrypt`

Reply:
I was confused at first by this since we don't support 32 bit...

But I see that the `MongoDB.Libmongocrypt.Test32` actually only tests that a `PlatformNotSupportedException` is thron.